src/app/wallets/index.types.d.ts

```diff
@@ -1,21 +1,21 @@
-type AddInvoiceSelfArgs = {
+type AddInvoiceArgs = {
```

Comment:
Since we are introducing `checkedXxx` conversion in the user case functions all the methods should accept `amount: number`.